src/events/guildMemberAdd.js

```diff
@@ -40,7 +40,14 @@ class GuildMemberAdd extends Event {
 
         if (settings.auto.nickname) member.setNickname(this.client.functions.formatMessage("autonick", guild, user, settings.auto.nickname)).catch(() => { return; });
 
-        const autorole = settings.auto.role.id ? guild.roles.has(settings.auto.role.id) ? guild.roles.get(settings.auto.role.id) : null : null;
+        let autorole;
+        if (settings.auto.role.bots && member.user.bot) {
+            autorole = settings.auto.role.bots ? guild.roles.has(settings.auto.role.bots) ? guild.roles.get(settings.auto.role.bots) : null : null;
+        }
+        else if (settings.auto.role.id) {
```

Comment:
Could you make this with the end bracket the same line to be consistent throughout the entire project.

Reply:
Updated